portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps.data;
```

Comment:
I like that you put this in its own file. Good job

Reply:
Thanks!